modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/ComposeContainerIT.java

```diff
@@ -0,0 +1,41 @@
+package org.testcontainers.junit.jupiter;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Testcontainers
+class ComposeContainerIT {
```

Comment:
I don't know how Testcontainers in general or these new tests in particular are structured, so my point may be moot, but here it goes: I moved away from using test class names to distinguish unit and integration test. Instead I apply Jupiter tags (e.g. with `@Tag("integration")`) and configure the build tool to filter by tags instead of file names. If you're interested in that, let me know - I have more than these two cents to give. :wink:

Reply:
We build Testcontainers using Gradle and currently don't have dedicated source sets for Unit- and Integration-Tests. I would assume if we would retstructure our testing phases, we would use two distinct source sets for Unit- and Integration-Tests, so we can use it for all modules.

Reply:
Alternatively, you could use tags and define two separate `test` tasks.

Reply:
I would avoid `IT` suffix (a rudiment from Maven)

Reply:
I think actually just copied over from the Spock tests :slightly_smiling_face: